monero-outreach-docs/translations/ru/storyboard/tool_for_renewal_ru.md

```diff
@@ -0,0 +1,77 @@
+# Инструмент перерождения
+**«Участвуйте в движении за позитивные изменения с помощью инструмента, который возвращает людям социальную и экономическую власть — Monero».**
+ 8 июня 2020 г.​
+
+На дворе 2020 год, и нами наблюдается многосторонний кризис. Глобальная пандемия, возобновление расовой и экономической напряжённости, крайне серьёзные геополитические разногласия и злоупотребление властью со стороны правительств и центральных банков по всему миру. Как подсказывает история, у нас есть все необходимые ингредиенты для того, чтобы произошли кардинальные изменения. Если вы поддерживаете тех, кто подвергается преследованиям по расовым, религиозным, политическим или экономическим причинам, или даже находитесь среди них, мы приглашаем вас принять участие в движении за позитивные перемены с помощью инструмента, который возвращает социальную и экономическую власть людям - Monero.
+
+Гражданские волнения и нарушение сложившегося порядка часто являются предвестниками глубоких изменений. Беспорядки в древнем Риме предвещали распад республики. В Америке независимость пришла после протестов, порчи имущества и насильственных ответных действий властей против протестующих (например, во время Бостонской резни). Протест и гражданское неповиновение сопровождали серьёзные изменения в гражданских правах начиная с 60-х годов прошлого века.
+
+Финансовые системы часто неразрывно связаны с происходящими беспорядками и социальными изменениями. Одной из причин американской войны за независимость стало колониальное стремление контролировать налогообложение и экономику в целом.
+
+Когда центральные банки начали печатать деньги после полной отмены золотого стандарта в 1971 году, появилась новая фиатная система (доллар США), которая стала глобальным монетарным ориентиром, определяющим стоимость большинства других валют. Теперь люди наблюдают непредвиденные последствия работы такой финансовой системы, а именно социально-экономическую хрупкость и неравенство. Им нужно что-то, что обеспечит более адекватный баланс власти. Но на этот раз нам не нужны революция и гражданская война для его достижения. Мы можем просто мирно выбрать технологическую финансовую систему, которая создается самими людьми и для них же самих.
+
+В 2009 году одним человеком или группой людей, выступившей под псевдонимом Сатоши Накамото, прямо по следам финансового кризиса был выпущен в свет предшественник Monero, названный Bitcoin. В генезис-блоке, первом блоке в блокчейне Bitcoin, Сатоши записал новостной заголовок, чтобы доказать время создания Bitcoin. Заголовок гласил: «The Times. 03 января 2009 г. Канцлер казначейства готов к повторному оказанию финансовой помощи банкам». Это оригинальное сообщение резонирует с сегодняшним днём, когда центральные банки стремятся поддержать мировую экономику, находящуюся в осаде.
```

Comment:
Hello @v1docq47,

I found a mistake in the English version, the first line of this paragraph, that can affect its meaning. Could you please check it out? The English version reads like this:

Wrong

- "In 2009 a pseudonymous person or group named Satoshi Nakamoto launched Monero’s precursor Bitcoin on the _**heals**_ of the great financial collapse."

Correct:

- "In 2009 a pseudonymous person or group named Satoshi Nakamoto launched Monero’s precursor Bitcoin on the _**heels**_ of the great financial collapse."

It was already fixed in PR #213.  

Thanks >:)

Reply:
@lh1008, tnx! everything is good :)